### PR TITLE
Feature/layout

### PR DESCRIPTION
**Pull request by kbardi**
- Improve components folder with each component in a different file.
- Adds Section layout component with Rows, Stack and Separator
- Adds progress bar component
- Adds TermsAndCondition component

### CODE REVIEW

File: src/lib/components/footer/footer.module.scss [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+  @media screen and (max-width: $lg) {
+    justify-content: flex-start;
+  }
+}
+
+.becomeFriend {
+  flex-direction: column;
+  display: flex;
+  width: 100%;
+  max-width: 350px;
```
Comment (davidpaley) [inline src/lib/components/footer/footer.module.scss]: we should use `em` here

File: src/lib/components/footer/footer.module.scss [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+}
+
+.formContainer {
+  margin-bottom: 2rem;
```
Comment (davidpaley) [inline src/lib/components/footer/footer.module.scss]: We say that we will use `rem` just for fonts. We should use `em` here

File: src/lib/components/progressBar/__snapshots__/progressBar.test.js.snap [hunk 3]
```diff
@@ -0,0 +1,112 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```
Comment (davidpaley) [inline src/lib/components/progressBar/__snapshots__/progressBar.test.js.snap]: We should put the files **/__sanpshots__/** in the gitignore, right?

File: src/lib/components/section/section.test.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Section from './section.js';
+
+describe('Section', () => {
+  it('renders properly without custom properties', () => {
+    const tree = renderer.create(<Section />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
```
Comment (davidpaley) [inline src/lib/components/section/section.test.js]: 👍

File: src/lib/components/progressBar/__snapshots__/progressBar.test.js.snap (hunk 3 above)
Comment (kbardi) [reply src/lib/components/progressBar/__snapshots__/progressBar.test.js.snap]: no, snapshots must be commited

**Pull request merged**